packages/api/src/command/medical/patient/get-quest-roster.ts

```diff
@@ -32,7 +34,7 @@ function getCommonQueryOptions({ pagination }: GetQuestRosterParams) {
               SELECT 1
               FROM patient_settings ps
               WHERE ps.patient_id = "PatientModelReadOnly"."id"
-              AND ps.subscriptions->'quest' IS NOT NULL
+              AND ps.subscriptions->'${settingsKey}' IS NOT NULL
```

Comment:
Are we updating existing data in the DB? How are we making this backwards compatible?